docs/documentation/release_notes/topics/22_0_0.adoc

```diff
@@ -84,4 +84,8 @@ The previous and now removed WildFly distribution provided a built-in vault prov
 
 = KeyStore Config Source added
 
-In relation to the KeyStore Vault news, we also integrated Quarkus's recently released feature called KeyStore Config Source. This means that among the already existing configuration sources (CLI parameters, environment variables and files), you can now configure your Keycloak server via configuration properties stored in a Java keystore file. You can learn more about this feature in the https://www.keycloak.org/server/configuration[Configuration guide].
\ No newline at end of file
+In relation to the KeyStore Vault news, we also integrated Quarkus's recently released feature called KeyStore Config Source. This means that among the already existing configuration sources (CLI parameters, environment variables and files), you can now configure your Keycloak server via configuration properties stored in a Java keystore file. You can learn more about this feature in the https://www.keycloak.org/server/configuration[Configuration guide].
```

Comment:
Hard to see what changed here, but did you mean to change this at all?

Reply:
It's just a white-space change. It can be safely ignored.